### PR TITLE
[1.6.1] Explicitly cast incorrect output type in OV model

### PR DESCRIPTION
**Pull request by sovrasov**
### Summary

This workarounds incorrect type of "labels" output in detection models on XPU

### CODE REVIEW

File: src/otx/algorithms/common/utils/ir.py [hunk 1]
```diff
@@ -32,6 +36,14 @@ def embed_ir_model_data(xml_file: str, data_items: Dict[Tuple[str, str], Any]) -
...
         model.set_rt_info(data, list(k))
 
+    # workaround for CVS-138901
+    if is_xpu_available():
+        ppp = PrePostProcessor(model)
+        for o in model.outputs:
+            if "labels" in o.get_names() and o.get_element_type() == Type.f32:
+                ppp.output("labels").tensor().set_element_type(Type.i64)
+        model = ppp.build()
+
```
Comment (jaegukhyun) [inline src/otx/algorithms/common/utils/ir.py]:
```suggestion
    # workaround for CVS-138901
    if is_xpu_available():
        pre_post_processor = PrePostProcessor(model)
        for output in model.outputs:
            if "labels" in output.get_names() and output.get_element_type() == Type.f32:
                pre_post_processor.output("labels").tensor().set_element_type(Type.i64)
        model = pre_post_processor.build()

```

**Comment by chuneuny-emily** [comment]
@sovrasov - Can you fix the failing issue?

**Comment by sovrasov** [comment]
> @sovrasov - Can you fix the failing issue?

I'll restart the CI, there's an instability in the runner

**Review by eunwoosh — APPROVED** [review]

File: src/otx/algorithms/common/utils/ir.py [hunk 2]
```diff
@@ -32,6 +36,14 @@ def embed_ir_model_data(xml_file: str, data_items: Dict[Tuple[str, str], Any]) -
     for k, data in data_items.items():
         model.set_rt_info(data, list(k))
 
+    # workaround for CVS-138901
```
Comment (kprokofi) [inline src/otx/algorithms/common/utils/ir.py:39]: Is it OK to write ticket number?

Comment (sovrasov) [reply src/otx/algorithms/common/utils/ir.py:39]: We had not reached any agreement on this yet, Emily to send the final guideline soon

**Review by kprokofi — APPROVED** [review]

**Comment by yunchu** [comment]
@sovrasov please don't forget to update changelog.md for your change.
@eunwoosh @kprokofi please don't forget to check whether changelog.md are updated as reviewer especially the target branch is release branch.

**Pull request merged**